packages/web-api-scan-runner/src/report-generator/axe-result-converter.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { AxeResults } from 'axe-core';
+import { ReportFormat } from 'storage-documents';
+
+export type ReportGenerationParams = {
+    pageTitle: string;
+    browserSpec: string;
+};
+
+export interface AxeResultConverter {
+    readonly reportType: ReportFormat;
```

Comment:
May be rename to resultReportFormat or targetReportFormat to indicate the converter target format operation